src/status.rs

```diff
@@ -140,6 +140,15 @@ pub struct RadioShow {
     pub end: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub enum IssueReportChannel {
+    Email,
+    IssueMail,
+    Twitter,
+    Ml,
+}
```

Comment:
The specs define the issue_report_channels as "array of strings" by switching to the enum we won't comply anymore.
Also this should include all possible report channels shouldn't it? So every possible value of the "contact" object?

Reply:
> The specs define the issue_report_channels as "array of strings" by switching to the enum we won't comply anymore.

Ah I'm wrong! Just the rendering on https://spacedirectory.org/pages/docs.html says its an array of strings. In the schema file itself its an enum with exactly those values!

Reply:
The Spacedirectory docs are incomplete unfortunately (we should fix that). If you look at archive.org:

https://web.archive.org/web/20160125010658/http://spaceapi.net/documentation

...you can see that allowed values are `email, issue_mail, twitter, ml`. So we're in compliance :)